libr/anal/dwarf_process.c

```diff
@@ -1182,86 +1178,48 @@ static st32 parse_function_args_and_vars(Context *ctx, ut64 idx, RStrBuf *args,
 						parse_abstract_origin (ctx, val->reference, &type, &name);
 						break;
 					case DW_AT_location:
-						if (val->kind == DW_AT_KIND_BLOCK) {
```

Comment:
why this removal?

Reply:
Because we can parse also a location list now, not just blocks (and the handling is inside the parse_location function)